torch/_inductor/codegen/wrapper.py

```diff
@@ -592,6 +594,23 @@ def generate(self, is_inference):
                     ),
                 ):
                     line.codegen(self.wrapper_call, device_cm_stack)
+                elif isinstance(line, OriginOpList):
+                    # Save the line containing OriginOp Info so
+                    # it can be added to the nvtx marker when the
+                    # the kernel launch instruction is generated.
+                    origin_info_list = line
+                elif (
+                    origin_info_list
+                    and isinstance(line, str)
+                    and not re.search(r"^#", line)
```

Comment:
It feels off to me that we should have to do a regex search in the middle of our code generation loop. Can we figure out a better way ?